arbitrum-docs/node-running/reference/software-releases.md

```diff
@@ -13,7 +13,13 @@ Arbitrum chains are powered by Arbitrum nodes running the Nitro software stack.
 
 Although Nitro is updated regularly, only a subset of Nitro releases carry ArbOS upgrades. These releases are significant because ArbOS upgrades are Arbitrum's equivalent to a hard fork - they can alter a node's ability to produce valid Arbitrum blocks. This is why validator nodes supporting a public Arbitrum chain (One, Nova) **must update Nitro** whenever a new ArbOS version is released.
 
-Note that every Nitro release is backwards compatible. In other words, the latest version of Nitro will support all previous ArbOS releases. This means that your validator's Nitro version must be greater than or equal to the version that includes the latest ArbOS upgrade.
+Note that every Nitro release is backwards compatible. In other words, the latest version of Nitro will support all previous ArbOS releases. This means that your validator's Nitro version must be greater than or equal to the version that includes the latest ArbOS upgrade. 
```

Comment:
If aligned on the above edit, we can use it here, too. Duplicated copy can be tough to maintain; partials solve that problem - I'd recommend using one here.

Reply:
Makes sense, i've added a partial and referenced it in both places!